src/drivers/deploy/drivers.yaml.template

```diff
@@ -54,6 +54,13 @@ spec:
           name: etc-path-ld
         - mountPath: /etc/docker
           name: etc-path-docker
+        - mountPath: /usr/local/nvidia # TODO
+          name: pre-install-nv-driver-path
+        env:
+        - name: DRIVER_PATH
+          value: /var/drivers/nvidia
+        - name: PRE_INSTALLED_NV_DRIVER_PATH
```

Comment:
This is assume deployer known this config in this file. If he's cluster contains driver in other path, he need to change this path config and deploy?

Reply:
Why not make this in service-configuration.yaml or kuberentes-configuration.yaml as a global configuration file config items?

Reply:
I will make this configurable after @ydye 's cluster object model merged into master. Right now, I think this is enough since we only need to cover aks case.

Reply:
ok, pls update doc in this place when it is configurable and tell user how to config
![image](https://user-images.githubusercontent.com/5576848/49125892-9af18500-f2fc-11e8-8d06-13e9ddaa0971.png)
https://github.com/Microsoft/pai/blob/master/docs/pai-management/doc/how-to-write-pai-configuration.md#gpu_driver

Reply:
Sure, I'll take a note.